providers/openfeature-provider-flagd/pyproject.toml

```diff
@@ -57,6 +57,11 @@ cov = [
   "cov-report",
 ]
 
+[tool.hatch.build.hooks.protobuf]
+dependencies = ["hatch-protobuf"]
+proto_paths = ["schemas"]
+output_path = "src/openfeature/contrib/provider/flagd"
```

Comment:
I put the schema directly into the root of src, because of problems with package imports of the generated files - I see you're affected by the same problem. I did not find a proper solution fastly, and thought this is a good compromise